.github/workflows/seldon_kind_test.yaml

```diff
@@ -0,0 +1,29 @@
+name: Build & Apply Seldon manifests in KinD
+on:
+  pull_request:
+    paths:
+      - contrib/seldon/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install KinD
+      run: ./tests/gh-actions/install_kind.sh
+
+    - name: Create KinD Cluster
+      run: ./tests/gh-actions/kind-cluster.yaml
```

Comment:
@axsaucedo I believe our issue is in this line, we'll need to run kind and use that file for creating the cluster.

The command should be
```bash
kind create cluster --config tests/gh-actions/kind-cluster.yaml
```
Similarly to https://github.com/kubeflow/manifests/blob/master/.github/workflows/jwa_kind_test.yaml#L20

Reply:
Oh my mistake, not sure how but I assumed this was an `.sh` script, but definitely need to grab a coffee

Reply:
Can definitely relate to that